### PR TITLE
[DCMTK] Update to 3.6.6

### PR DESCRIPTION
**Pull request by BigSeb**
**Describe the pull request**

- #### What does your PR fix?  
  Update DCMTK to 3.6.6

- #### Which triplets are supported/not supported? Have you updated the [CI baseline]
No change of triplets have been made, it should be the same as the previous version. It was personally tested with x86-windows and x64-windows.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/dcmtk/dcmtk.patch [hunk 1]
```diff
@@ -1,16 +1,16 @@
-diff --git a/CMake/dcmtkPrepare.cmake b/CMake/dcmtkPrepare.cmake
-index e9f77b5..4f7ebe3 100644
```
Comment (ras0219-msft) [inline ports/dcmtk/dcmtk.patch:2]: Is the change to this patch file needed? It looks like you've just uppercased the resulting `set` directive.

Comment (BigSeb) [reply ports/dcmtk/dcmtk.patch:2]: Line numbers where the patch must be applied have been changed. I have generated a new patch using git in order to generate this new patch version. I did not change the original case of the CMake code (but the creator of the patch did change the case). I agree that case of the command is not important for the patch, only the context in which the patch must be applied need to change (line numbers).

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks for the PR!

**Comment by strega-nil-ms** [comment]
Cool, thanks @BigSeb :)

**Pull request merged**